modules/search/instant-search/components/search-results.jsx

```diff
@@ -23,6 +23,9 @@ class SearchResults extends Component {
 		const hasCorrectedQuery = corrected_query !== false;
 		const num = new Intl.NumberFormat().format( total );
 
+		if ( this.props.isLoading ) {
+			return sprintf( __( 'Looking for "%s"', 'jetpack' ), this.props.query );
```

Comment:
This needs to handle the case when the query string is empty. Maybe just show "Looking..." in that case.

Reply:
I've added some copy suggestions here too: https://github.com/Automattic/jetpack/issues/14782#issuecomment-594032689

Reply:
Pushed out a copy update with 03fef86.